test_launch_ros/test/test_launch_ros/actions/test_load_composable_nodes.py

```diff
@@ -388,12 +388,12 @@ def test_load_node_with_param_file(mock_component_container):
     assert request.node_name == 'node_1'
     assert request.node_namespace == '/ns_1'
     assert len(request.parameters) == 3
-    assert request.parameters[0].name == 'param_2'
-    assert request.parameters[0].value.integer_value == 2
-    assert request.parameters[1].name == 'param_3'
-    assert request.parameters[1].value.integer_value == 33
-    assert request.parameters[2].name == 'param_1'
-    assert request.parameters[2].value.integer_value == 1
+    assert request.parameters[0].name == 'param_1'
+    assert request.parameters[0].value.integer_value == 1
+    assert request.parameters[1].name == 'param_2'
+    assert request.parameters[1].value.integer_value == 2
+    assert request.parameters[2].name == 'param_3'
+    assert request.parameters[2].value.integer_value == 33
```

Comment:
I think we can also add the test case for `/*` for `example_parameters_multiple_entries.yaml`.